src/librepgp/stream-armor.cpp

```diff
@@ -676,6 +686,14 @@ armor_parse_headers(pgp_source_t *src)
             RNP_LOG("Too long armor header - truncated.");
             header[hdrlen] = '\0';
         } else if (hdrlen) {
+            if (is_base64_line(header, hdrlen)) {
+                RNP_LOG("Warning: no empty line after the base64 headers");
+                if (param->type == PGP_ARMORED_CLEARTEXT) {
```

Comment:
Hm, despite approval, we should probably cover these lines with tests? And why this should fail here? Because of absence of Hash: header?

Reply:
https://github.com/rnpgp/rnp/issues/1289#issuecomment-763972963

Reply:
Thanks, got it. Anyway, could you please add some small test for this case to make codecov happy? It can be done by modifying `message.txt.cleartext-signed` and adding test to `cli_tests.py` or updating `test_ffi_op_verify_sig_count`.

Reply:
looks line this condition is not needed here at all, cleartext signatures are parsed in other place, `cleartext_parse_headers()` in `stream-parse.cpp`

Reply:
more precisely, `is_base_64_line()` can return false, as cleartext message can contain any characters outside of Base64 alphabet, so execution does not reach this condition 
`if (param->type == PGP_ARMORED_CLEARTEXT) {`

Reply:
Agree, feel free to remove that condition.